tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/experiment_builder.py

```diff
@@ -370,7 +335,7 @@ def populate_presence_matrix(self, datasets: List[Dataset]) -> None:
             max_dataset_joinid = max(d.soma_joinid for d in datasets)
 
             # LIL is fast way to create spmatrix
-            pm = sparse.lil_array((max_dataset_joinid + 1, self.n_var), dtype=bool)
+            pm = sparse.lil_matrix((max_dataset_joinid + 1, self.n_var), dtype=bool)
```

Comment:
They are almost aliases, but not quite - they have slightly different semantics, and the `_array` variant is preferred for new code.  See the huge note at the top of the docs [here](https://docs.scipy.org/doc/scipy/reference/sparse.html).

The difference doesn't matter for this use case so lets stick with the modern API

Reply:
yes, correct - for consistency with the rest of the builder which uses `_matrix`.  The difference is inconsequential in this case - see doc link.  Historically single cell always uses matrix because community use (e.g., AnnData) predates the arrival of `_array`